README.md

```diff
@@ -65,15 +27,13 @@ After you're finished please remove all the comments and instructions!
 
 # 📖 [your_project_name] <a name="about-project"></a>
```

Comment:
- [ ] You have done incredibly well almost implementing all required changes but a key part is still missing, and that is the project name which is more like a hyperlink here. Kindly customize this to reflect the appropriate title of your project and to also ensure that users understand or feel what your project is about. Here's where the previous reviewer requested it:
![image](https://github.com/diarisdiakite/vet-clinic/assets/99715640/a6577f6c-189b-456e-b565-352b4ef12c08)